doc/variables.md

```diff
@@ -99,7 +99,13 @@ For more details see [scenario documentation](/doc/scenario.md).
 * `all_rw` (`boolean`): indicates that that all servers in the replicaset should be read-write;
 * `vshard_group` (`string`): name of vshard group the replicaset belongs to;
 * `weight` (`number`): vshard replicaset weight (matters only if `vshard-storage` role is enabled);
-* `edit_topology_timeout` (`number`, default: `60`): time in seconds to wait until a cluster become
+* `edit_topology_netbox_call_timeout` (`number`): time in seconds to wait netbox call
```

Comment:
Not sure that `edit_topology_` prefix is requred. Seems that `vars.options` affects everything, am I right?